week-2/Homework/mandatory/3-playing-computer.js

```diff
@@ -40,3 +40,16 @@ for (let i = 0; i < 5; ++i) {
     console.log(e);
   }
 }
+
+
+//                    Values
+//-------------------------------------------------------
+//   i   |   a   |   d   |   e   |   f1   |   f2   |
+//-------------------------------------------------------
+//   0   |   7   |   4   |   -   |   --   | 1 time |
+//   1   |   8   |       |   9   | 1 time |   --   |
+//   2   |   9   |   6   |   -   |   --   | 1 time |
+//   3   |   10  |       |   13  | 1 time |   --   |
+//   4   |   11  |   8   |   -   |   --   | 1 time |
+//--------------------------------------------------------
+//   5   |   11  |   8   |   13  | 2 times| 3 times|
```

Comment:
Amazing! 💯 This is an excellent way to keep track of what's going on! Really impressive 🥇